hieradata/site/ls/cluster/lsstcam-ccs.yaml

```diff
@@ -0,0 +1,122 @@
+---
+profile::nm::connections:
```

Comment:
I would probably merge this commit into a6d4199d2bafeffc58059e7f76bb9d8cfbcd54b2 or reorder the PR so that this change is in the history prior to the nodes that require it. Merging is probably slightly better as that would put spec coverage of the cluster into the same commit.

Reply:
This is still pending.